tests/uber/conftest.py

```diff
@@ -87,7 +87,7 @@ def init_db(request):
     patch_session(Session, request)
     initialize_db(modify_tables=True)
     register_session_listeners()
-    with Session() as session:
+    with Session(no_cache=True) as session:
```

Comment:
The session used to set up the database needs to be different from the one used by the tests.